.github/workflows/continuous-deploy.yml

```diff
@@ -239,10 +246,20 @@ jobs:
           name: coverage-files-${{ matrix.module.id }}
           path: coverage-files/.coverage.${{ matrix.module.id }}
 
+  status-check:
```

Comment:
As far as I remember, if any step within a matrix fails (by default) then all steps that have not yet completed are automatically canceled and all subsequent steps are skipped. 

Is this status check step required?